### PR TITLE
[#55016] Timeout open project storage modal after 120 secs.

### PR DESCRIPTION
**Pull request by ba1ash**
https://community.openproject.org/work_packages/55016

image
image

@wielinde have a look at the behavior, please.

### CODE REVIEW

File: config/locales/js-en.yml [hunk 1]
```diff
@@ -1447,6 +1447,11 @@ en:
...
         Close: "Close"
     open_project_storage_modal:
+      waiting_title:
+        timeout: "Timeout"
       waiting_subtitle:
         network_off: "There is a network problem."
         network_on: "Network is back. We are trying."
+        timeout: >
+          OpenProject could not provide you access to the project folder within the expected period of time. Please, try once again.<br><br>
+          If that problem persists please contact you OpenProject administrator to check the <a href="https://www.openproject.org/docs/system-admin-guide/file-storages/file-storage-troubleshooting/">health status of the file storage setup</a>.
```
Comment (dominic-braeunlein) [inline config/locales/js-en.yml]: Hi @ba1ash, the link changed to https://www.openproject.org/docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/

**Review by dominic-braeunlein — CHANGES_REQUESTED** [review]
LGTM 👍
The one URL changed in the meanwhile

**Comment by dominic-braeunlein** [comment]
Looks good 👍

**Review by dominic-braeunlein — APPROVED** [review]
👍

**Comment by mMuck** [comment]
Don't know if this is the right place but there is a typo in line https://github.com/opf/openproject/blob/d231bbbe11e95045770ef1e704a1a7aab6706fe3/config/locales/js-en.yml#L1453

> please contact you_**r**_ OpenProject administrator

**Comment by ba1ash** [comment]
> Don't know if this is the right place but there is a typo in line
> 
> https://github.com/opf/openproject/blob/d231bbbe11e95045770ef1e704a1a7aab6706fe3/config/locales/js-en.yml#L1453
> 
> > please contact you_**r**_ OpenProject administrator

Thanks for pointing it out @mMuck!
Yes, it is not the right place. The ideal place would be to report it in https://community.openproject.org/ as  a bug.

**Pull request merged**